karma.conf.js

```diff
@@ -55,14 +55,7 @@ module.exports = function(config) {
     },
 
     browserify: {
-        debug: true,
-        configure: function(bundle) {
-            bundle.on('prebundle', function() {
-                // Exclude jsdom because it makes IE =< 8 go kaboom.
-                // And browsers have real DOM, right?
-                bundle.exclude('jsdom');
-            });
-        }
+        debug: true
```

Comment:
Except for `debug`, this is handles by the `browserify` config in `package.json`.